pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotBrokerDebug.java

```diff
@@ -102,15 +102,15 @@ public Map<String, Map<ServerInstance, List<String>>> getRoutingTable(
     if (tableType != TableType.REALTIME) {
       String offlineTableName = TableNameBuilder.OFFLINE.tableNameWithType(tableName);
       RoutingTable routingTable = _routingManager.getRoutingTable(
-          CalciteSqlCompiler.compileToBrokerRequest("SELECT * FROM " + offlineTableName));
+          CalciteSqlCompiler.compileToBrokerRequest("SELECT * FROM " + offlineTableName), 0);
```

Comment:
IIRC, presto connector relies on this to return the routing table. Let's add a `requestIdGenerator` in this class to avoid always using the same request id.